pkg/ssh/exec.go

```diff
@@ -184,6 +187,60 @@ More information is available here: https://okteto.com/docs/reference/manifest#r
 	return err
 }
 
+func waitForOtherCopyToFinish() {
+	wasRunning := false
+Loop:
```

Comment:
is it possible to avoid this syntax and use a structured loop instead?

Reply:
yes, we could get out of the loop with a return statement and not use the tag